File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> milo = new HashSet<>();
        for (int num : nums) {
            if (milo.contains(num)) {
                return true;
            }
            milo.add(num);
        }
        return false;
    }
}

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res = [set()]
        for num in nums: res.append({num})
        for i in range(len(res)):
            for j in range(len(res)):
                if res[i] | res[j] not in res: res.append(res[i] | res[j])
        return [list(x) for x in res]